src/Backend/Core/Layout/Templates/FormLayout.html.twig

```diff
@@ -374,3 +380,13 @@
     {% endif %}
   </div>
 {% endblock %}
+
+{% block help %}
+  {% for attributeName, attributeValue in attr %}
+    {% if attributeName == 'help' %}<span class="help-block">{{ attributeValue|trans }}</span>{% endif %}
```

Comment:
you can now have 2 help blocks by doing it like this if you set one from the template and one from the php can you fix this please, also if possible, avoid the duplicate html

Reply:
Yes indeed, that is possible.
Is that a problem?

Reply:
why would you even allow that?
I don't mind that you can overwrite it from the template, but this is a very strange implementation

Reply:
So you prefer the "attr.help"? And not the variable "help"?

Reply:
indeed